lld/wasm/SyntheticSections.cpp

```diff
@@ -514,7 +514,10 @@ void GlobalSection::writeBody() {
     } else {
```

Comment:
does the extended-const case also need to be updated?

Reply:
No since in that case we do the addition with `__tls_base` as part of the const expression.  i.e. its don't for us at instantiation time.

So I guess there are 3 ways this addition can happen:

1. At instantiation time via extended const
2. As part of __wasm_apply_global_tls_relocs at runtime
3. At static link time via `getVA(absolute=true)`